src/pages/Sources.js

```diff
@@ -185,8 +186,8 @@ const SourcesPage = () => {
   const setSelectedType = (selectedType) => stateDispatch({ type: 'setSelectedType', selectedType });
 
   const mainContent =
-    activeCategory === INTEGRATIONS && enableIntegrations ? (
-      <AsyncComponent appName="notifications" module="./IntegrationsTable" />
+    [INTEGRATIONS, COMMUNICATIONS, REPORTING, WEBHOOKS].includes(activeCategory) && enableIntegrations ? (
+      <AsyncComponent appName="notifications" module="./IntegrationsTable" activeCategory={activeCategory} />
```

Comment:
Same as above, we probably want to fallback to previous implementation if `platform.sources.breakdown` is disabled.

Reply:
Tested again and I hope it works this way now